pkg/cloudevents/client/client.go

```diff
@@ -107,6 +111,15 @@ func (c *ceClient) obsSend(ctx context.Context, event cloudevents.Event) (contex
 		}
 	}
 
+	// Set distributed tracing extension.
+	if !c.disableTracePropagation {
+		if span := trace.FromContext(ctx); span != nil {
```

Comment:
So spans are only propagated if they exist on the context, correct?

Reply:
Correct, although if tracing is enabled in the SDK a new trace will be created immediately before this.